src/cmake/STIRConfig.cmake.in

```diff
@@ -143,6 +143,11 @@ if(@STIR_WITH_NiftyPET_PROJECTOR@)
   set(STIR_WITH_NiftyPET_PROJECTOR TRUE)
 endif()
 
+if(@STIR_WITH_STIR_CUDA@)
+  find_package(CUDAToolkit REQUIRED)
```

Comment:
possbily needs `enable_language(CUDA)` as well, not sure

Reply:
done here: https://github.com/UCL/STIR/pull/1408/commits/d3f37324f65127673b04bfbaad1952c470edfab5